include/picongpu/simulation/stage/CurrentInterpolationAndAdditionToEMF.hpp

```diff
@@ -127,6 +127,16 @@ namespace picongpu
                             fieldSolver.addCurrent<type::CORE + type::BORDER>();
                         }
                     }
+                    else
+                    {
+                        /* With no current from macroparticles, there is no need for communication.
```

Comment:
Let us check this tomorrow together. Field background is only created on `CORE+BORDER` therefore we need to read from the neighbors to have valid data in the guard for the binomial smoothing operation.
Nevertheless, we need to avoid scattering guard data twice into the border, we only need to read border data from neighbors into the guard.
I think 4 eyes should avoid introducing a new bug.

Reply:
Oh true, let's discuss tomorrow.

Reply:
I think what you describe is another bug that is present regardless of thus PR

Reply:
Discussed offline, summarized in #4250 . Unrelated to this PR.